index.php

```diff
@@ -142,6 +142,12 @@ function get_q_value($mime, $accept)
         error_page('Verification Failed', 'Given Code Was Invalid');
     }
 
+    $response = array('me' => USER_URL);
+
+    if ($scope = (explode(':', $code, 3))[2]) {
```

Comment:
Actually, the extra brackets seem to break in PHP versions below 7.0.

In 5.6.29 I get an “syntax error, unexpected '['” on `(explode(':', $code, 3))[2]`. Just `explode(':', $code, 3)[2]` will work back to 5.4.0.

I propose just not trying to do it on one line, it will spare us the headache.